static/app/views/dashboardsV2/widgetCard/chart.tsx

```diff
@@ -318,6 +326,7 @@ class WidgetCardChart extends React.Component<WidgetCardChartProps, State> {
         }
         return seriesName;
       },
+      ...(legendOptions ?? {}),
```

Comment:
```suggestion
      ...legendOptions,
```

Expanding a null or undefined adds nothing to an object, so no need to default to an empty object